src/hooks/useRoomList.ts

```diff
@@ -0,0 +1,36 @@
+import { useMemo } from "react";
+
+import { Room } from "types/rooms";
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+
+import { useRelatedVenues } from "hooks/useRelatedVenues";
+import { useRecentLocationUsers } from "hooks/users";
+
+export interface useRoomListProps {
+  roomList?: Room[];
+}
+
+export const useRoomList = ({ roomList }: useRoomListProps) => {
+  const roomUrls = roomList?.map(
+    (room) => room?.title.toLowerCase().replaceAll(" ", "") ?? ""
+  );
```

Comment:
is something changed and now it's a correct way of getting room URL?
because previously we got URL from `room.url` field, isn't it?

@mike-lvov, can you take a look at this part, please?

Reply:
https://github.com/sparkletown/sparkle/pull/1983#issuecomment-903551700

Reply:
> Looks to me like this [#1983 (comment)](https://github.com/sparkletown/sparkle/pull/1983#discussion_r690403604) is a valid concern to be addressed before continuing. So far rooms have only been accessed through index and even if string key is employed, it should be surrogate, not the title

Agree, working on it now

Reply:
fixed 845647c